### DISCUSSION TITLE
AbortPlant and AbortDefuse not working

### DISCUSSION THREAD

**Discussion by iamleniac** [saul/demofile#250] [Questions] [answered]
I'm trying to use these two events to run data analysis, but they are'nt working on any demo I've tested on.

**Comment by saul** [comment, marked as answer]
A lot of events aren't actually recorded to demos to save network traffic.

You can instead get what you want by listening to when DT_CSPlayer.m_bIsDefusing changes. You can tell if a defuse has been aborted by checking when it changes from 1 -> 0 (and also that the bomb is not yet defused).

As for plant aborting, you might want to check some of the entity properties on DT_WeaponC4. Maybe look at the m_bStartedArming property.

Let me know if you're having trouble.

**Comment by iamleniac** [comment]
I can get it using the entity changes you suggested, thanks for that. Do you know if is there a way to get the tactical timeouts? It would be great if I could get the round number where one of the teams called a timeout.

**Comment by saul** [comment]
What do you mean by tactical timeout? If it's a separate question, please can you open a new issue? That way it will be easier to search for users who may have a similar question in future. Thanks
